queries/subsquid/general/allEventsByProfile.graphql

```diff
@@ -0,0 +1,22 @@
+query allEventsByProfile($id: String) {
```

Comment:
here we go again hehe

Reply:
it's really similar query to the one that was previously used in profile page:
`rmrk/subsquid/allEventsByProfile.graphql`